### PR TITLE
Lesson incubation proposal

### PR DESCRIPTION
**Pull request by gvwilson**

### CODE REVIEW

**Comment by jduckles** [comment]
Lets drop the menu link, we already have too many menu options. A link from the lessons page should suffice.

**Comment by gvwilson** [comment]
PR updated.

**Comment by jduckles** [comment]
Ready to merge?

**Comment by gvwilson** [comment]
Waiting to hear from @tracykteal about publishing it on the DC site 
simultaneously.

File: pages/lessons_incubation.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+9.  Early adopters (and the lesson leads) deliver the lesson at
+    multiple sites, coalescing feedback and refining the lesson.
+
+10. The lesson leads and Carpentry community members then publish the
+    lesson on the Carpentry websites.
+
+11. The lesson leads provide two hour-long online training sessions
+    in which they discuss the motivation for the lesson, their
+    experience developing it, and what they have learned from early
+    deliveries.
```
Comment (wking) [inline pages/lessons_incubation.md:158]: Who are these training sessions delivered to?  Where are they advertized?  Is it “advertized on discuss@ (or the analagous DC channel) for attendance by the general public but pitched for existing SWC(/DC) instructors”?  Or is it pitched at existing maintainers?  Or…?

File: pages/lessons_incubation.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+    sponsoring organization.
+
+4.  If a full two-day workshop is being developed, the budget must
+    also include travel money for a hackathon at which early adopters
+    (both those at trial sites and others) can work together to refine
+    the first version based on early experience teaching it. This is
+    usually a 3-day event for 10–15 people whose costs to participate
+    are covered, at which the lead developer(s) will usually teach the
+    entire material for early adopters to observe. Based on Data
+    Carpentry’s experience, such a hackathon costs roughly US$20,000.
```
Comment (wking) [inline pages/lessons_incubation.md:99]: This seems like a big ask ;).  Have any existing SWC lessons had a three-day, $20k hackathon with 10-15 people meeting in person to work on the lesson?

**Pull request merged**